webview/src/experiments/components/table/body/TableContent.test.tsx

```diff
@@ -19,210 +22,64 @@ jest.mock('../../../../shared/api')
-    getRowModel: () => ({
-      flatRows: [
-        {
-          columnFilters: {},
-          columnFiltersMeta: {},
-          depth: 0,
-          id: '1',
-          index: 1,
-          original: {
-            Created: '2023-04-20T05:14:46',
-            branch: 'main',
-            commit: {
-              author: 'Matt Seddon',
-              date: '31 hours ago',
-              message: 'Update dependency dvc to v2.55.0 (#76)\n\n',
-              tags: []
-            },
-            deps: {
-              data: {
-                changes: false,
-                value: 'ab3353d'
-              },
-              'train.py': {
-                changes: false,
-                value: 'f431663'
-              }
-            },
-            description: 'Update dependency dvc to v2.55.0 (#76)',
-            id: 'a9b32d1',
-            label: 'a9b32d1',
-            metrics: {
-              'training/metrics.json': {
-                step: 14,
-                test: {
-                  acc: 0.7735,
-                  loss: 0.9596208930015564
-                },
-                train: {
-                  acc: 0.7694,
-                  loss: 0.9731049537658691
-                }
-              }
-            },
-            params: {
-              'params.yaml': {
-                epochs: 15,
-                lr: 0.003,
-                weight_decay: 0
-              }
-            },
-            selected: false,
-            sha: 'a9b32d14966b9be1396f2211d9eb743359708a07',
-            starred: false,
-            subRows: [
-              {
-                Created: '2023-04-21T12:04:32',
-                deps: {
-                  data: {
-                    changes: false,
-                    value: 'ab3353d'
-                  },
-                  'train.py': {
-                    changes: false,
-                    value: 'f431663'
-                  }
-                },
-                description: '[prize-luce]',
-                id: 'prize-luce',
-                label: 'ae4100a',
-                metrics: {
-                  'training/metrics.json': {
-                    step: 14,
-                    test: {
-                      acc: 0.7735,
-                      loss: 0.9596208930015564
-                    },
-                    train: {
-                      acc: 0.7694,
-                      loss: 0.9731049537658691
-                    }
-                  }
-                },
-                params: {
-                  'params.yaml': {
-                    epochs: 15,
-                    lr: 0.003,
-                    weight_decay: 0
-                  }
-                },
-                selected: false,
-                sha: 'ae4100a4b4a972c3ceffa3062739845d944b3ddf',
-                starred: false
-              }
-            ]
+const mockedGetIsExpanded = jest.fn()
+const mockedGetAllCells = jest.fn().mockReturnValue([1, 2, 3, 4, 5]) // Only needed as length
+const mockRowModel = {
```

Comment:
[Q] Can we move this out of this file so that any further changes to this fixture aren't mixed in with test changes?

Reply:
Moved into a separate file!